decidim-core/app/helpers/decidim/decidim_form_helper.rb

```diff
@@ -201,7 +201,7 @@ def decidim_form_slug_url(prepend_path = "", value = "")
     #
     # Returns an HTML-safe String.
     def form_required_explanation
-      content_tag(:div, class: "help-text help-text-form-required-fields") do
+      content_tag(:div, class: "text-sm text-gray-2") do
```

Comment:
This kind of stuff becomes really hard to customize and maintain the customizations. This is my never ending issue with the Tailwind... :worried: